doozer/doozerlib/cli/images_konflux.py

```diff
@@ -188,3 +196,192 @@ async def images_konflux_build(
         konflux_context=konflux_context, konflux_namespace=konflux_namespace,
         image_repo=image_repo, skip_checks=skip_checks, dry_run=dry_run)
     await cli.run()
+
+
+class KonfluxBundleCli:
+    def __init__(
+        self,
+        runtime: Runtime,
+        operator_nvrs: Sequence[str],
+        force: bool,
+        dry_run: bool,
+        konflux_kubeconfig: Optional[str],
+        konflux_context: Optional[str],
+        konflux_namespace: str,
+        image_repo: str,
+        skip_checks: bool,
+        release: Optional[str],
+    ):
+        self.runtime = runtime
+        self.operator_nvrs = list(operator_nvrs)
+        self.force = force
+        self.dry_run = dry_run
+        self.konflux_kubeconfig = konflux_kubeconfig
+        self.konflux_context = konflux_context
+        self.konflux_namespace = konflux_namespace
+        self.image_repo = image_repo
+        self.skip_checks = skip_checks
+        self.release = release
+
+    @start_as_current_span_async(TRACER, "images:konflux:bundle")
+    async def run(self):
```

Comment:
I suggest splitting it into smaller functions to enhance readability and maintainability